src/main/kotlin/org/opensearch/replication/metadata/UpdateReplicationMetadata.kt

```diff
@@ -49,29 +49,34 @@ class UpdateReplicationStateDetailsTaskExecutor private constructor()
 
     override fun execute(currentState: ClusterState, tasks: List<UpdateReplicationStateDetailsRequest>)
             : ClusterStateTaskExecutor.ClusterTasksResult<UpdateReplicationStateDetailsRequest> {
-        return getClusterStateUpdateTaskResult(tasks[0], currentState)
+        log.info("Executing replication state update for $tasks")
```

Comment:
Can this get noisy?

Reply:
we are using this only during index's replication state transition and logged only when we are moving to running, pause, resume or stopped.
but do let me know, If it is not adding any value, I can move to debug level as well.